index.css

```diff
@@ -0,0 +1,201 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  background-color: #eee;
+  width: 100%;
+  height: 100%;
+}
+
+a:link,
+a:hover {
+  text-decoration: none;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #eee;
+  position: fixed;
+  top: 0%;
+  right: 0%;
+  height: 65px;
+  width: 100%;
+  z-index: 5;
+}
+
+.navbar .navbar__brand {
+  width: 300px;
+  margin-left: 30px;
+}
+.navbar .navbar__brand img {
+  width: 100%;
+}
+
+.navbar .navbar__links ul {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+}
+.navbar .navbar__links ul li a {
```

Comment:
- [ ] Kindly ensure you add a new line here too. An empty new line after each css block